flink-runtime/src/test/java/org/apache/flink/runtime/testutils/CommonTestUtils.java

```diff
@@ -367,9 +367,11 @@ public static void waitForCheckpoint(JobID jobID, MiniCluster miniCluster, int n
                 });
     }
 
-    /** Wait for on more completed checkpoint. */
-    public static void waitForOneMoreCheckpoint(JobID jobID, MiniCluster miniCluster)
-            throws Exception {
+    /**
+     * Wait for a new completed checkpoint. Note: we wait for 2 or more checkpoint to ensure the
+     * latest checkpoint start after waitForTwoOrMoreCheckpoint is called.
+     */
+    public static void waitForNewCheckpoint(JobID jobID, MiniCluster miniCluster) throws Exception {
```

Comment:
I have refactored the `waitForNewCheckpoint`. I check the checkpoint trigger time instead of checkpointCount.

I think checking trigger time is clearer than checkpointCount >= 2, WDYT?